### PR TITLE
remove unnecessary dependencies

### PR DESCRIPTION
**Pull request by spalger**
I'm trying to remove expect.js from my project and noticed it was being installed by this package (transitive dep of mapbox-gl)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,10 +10,7 @@
     "sharkdown": "sharkdown"
   },
   "dependencies": {
-    "expect.js": "~0.2.0",
```
Comment (tmcw) [inline package.json]: These shouldn't be dependencies, but they need to be devDependencies - the tests rely on them.

Comment (spalger) [reply package.json]: Expect and stream-spigot already are, but I couldn't find any references to through

Comment (tmcw) [reply package.json]: Okay, yeah - so through should be removed.

**Comment by spalger** [comment]
Based on your most recent comment, this should be good to go right?

**Pull request merged**